### DISCUSSION TITLE
Fix backlogs from v1

### DISCUSSION THREAD

**Discussion by sak007** [sak007/MyDollarBot-BOTGo#19] [Announcements]
## What's Changed
* Refactored code.py and moved functions to different files based on command
* Fixed edit command
* Added missing badges
* Added code coverage
* Added test cases for existing functions

## New Contributors
* @zedunaid @sak1997 @wizard-31 @rjain09 @sak007   made their first contribution in https://github.com/sak007/MyDollarBot/pull/16

**Full Changelog**: https://github.com/sak007/MyDollarBot/commits/v1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/sak007/MyDollarBot/releases/tag/v1.1'>Fix backlogs from v1</a>.</em>

**Comment by wizard-31** [comment]
Looks like we have a ~80% code coverage with 4 modules. Let's try to improve in the next release. 
Reviewers will not accept PR if codecov reduces than current value.

**Reply by sak1997** [reply]
Understood. Let's look at which modules are not complying with our chosen metric and work on that.
